v3/plugins/redhat/quarkus-java11/1.6.0/meta.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+publisher: redhat
+name: quarkus-java11
+version: 1.6.0
+type: VS Code extension
+displayName: Quarkus Tools with Java 11
+title: Quarkus Tools by Red Hat
+description: This plug-in provides support for Quarkus development via a Quarkus language server, Java jdt.ls and Java Debugger.
+icon: https://raw.githubusercontent.com/redhat-developer/vscode-quarkus/v1.6.0/icons/logo.png
+repository: https://github.com/redhat-developer/vscode-quarkus
+category: Language
+firstPublicationDate: "2020-08-18"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11-f76ca45"
```

Comment:
Please use the latest che-sidecar-java:11 image listed in the [vscode-extension.json file](https://github.com/eclipse/che-plugin-registry/blob/master/vscode-extensions.json) for the quarkus extension.